picky-asn1-der/src/application_tag.rs

```diff
@@ -0,0 +1,158 @@
+use crate::misc::Length;
+use crate::Asn1RawDer;
+use picky_asn1::tag::{Tag, TagPeeker};
+use serde::de::{Error, SeqAccess};
+use serde::{de, ser};
+use std::fmt;
+use std::fmt::Debug;
+
+#[derive(Debug, PartialEq)]
+pub struct ApplicationTag<V: Debug + PartialEq, const T: u8>(V);
```

Comment:
I think we already bumped the minimal required rustc version for the next release, but don't forget to take this into account when using latest features :slightly_smiling_face: